api/src/main/java/org/openmrs/module/fhirExtension/translators/impl/ObsBasedDiagnosticReportTranslatorImpl.java

```diff
@@ -32,16 +38,23 @@ public class ObsBasedDiagnosticReportTranslatorImpl implements ObsBasedDiagnosti
 	@Autowired
 	private DiagnosticReportObsLabResultTranslatorImpl diagnosticReportObsLabResultTranslator;
 	
+	@Autowired
+	private ProviderService providerService;
+	
+	static final String PRACTITIONER = "Practitioner";
+	
 	@Override
 	public DiagnosticReport toFhirResource(@Nonnull FhirDiagnosticReport fhirDiagnosticReport) {
 		DiagnosticReport diagnosticReport = diagnosticReportTranslator.toFhirResource(fhirDiagnosticReport);
 		setPresentedForm(diagnosticReport, fhirDiagnosticReport);
+		setPractitioner(diagnosticReport, fhirDiagnosticReport);
```

Comment:
There is already one translator in fhir2 codebase: PractitionerReferenceTranslatorProviderImpl.
Can we reuse the same for the purpose.

Reply:
Changes are made